### PR TITLE
fix: failed to new piece store will panic

### PR DESCRIPTION
**Pull request by sysvm**
### Description

add a description of your changes here...

### Rationale

N/A

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

**Review by constwz — APPROVED** [review]
LGTM

**Review by BarryTong65 — APPROVED** [review]

File: base/gfspapp/app_options.go [hunk 1]
```diff
@@ -354,27 +353,39 @@ func DefaultGfSpPieceStoreOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) e
...
+		}
+		if cfg.PieceStore.Store.MinRetryDelay == 0 {
+			cfg.PieceStore.Store.MinRetryDelay = 1
+		}
+		if cfg.PieceStore.Store.IAMType == "" {
+			cfg.PieceStore.Store.IAMType = "SA"
+		}
+		pieceStore, err := piecestoreclient.NewStoreClient(&cfg.PieceStore)
+		if err != nil {
+			log.Errorw("if not use piece store, please ignore: failed to new piece store", "error", err)
```
Comment (will-2012) [inline base/gfspapp/app_options.go]: Refine this log.

File: base/gfspapp/app_options.go [hunk 2]
```diff
@@ -354,27 +353,39 @@ func DefaultGfSpPieceStoreOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) e
...
+		if cfg.PieceStore.Store.MaxRetries == 0 {
+			cfg.PieceStore.Store.MaxRetries = 5
+		}
+		if cfg.PieceStore.Store.MinRetryDelay == 0 {
+			cfg.PieceStore.Store.MinRetryDelay = 1
+		}
+		if cfg.PieceStore.Store.IAMType == "" {
+			cfg.PieceStore.Store.IAMType = "SA"
+		}
+		pieceStore, err := piecestoreclient.NewStoreClient(&cfg.PieceStore)
```
Comment (will-2012) [inline base/gfspapp/app_options.go:381]: sync.Once?

File: base/gfspapp/app_options.go (hunk 1 above)
Comment (sysvm) [reply base/gfspapp/app_options.go]: fixed

File: base/gfspapp/gfsp_modmgr.go [hunk 3]
```diff
@@ -83,7 +83,7 @@ func GetNewModularFunc(name string) NewModularFunc {
 	modMgr.mux.RLock()
 	defer modMgr.mux.RUnlock()
 	if _, ok := modMgr.newModularFunc[name]; !ok {
-		log.Panicf("not register [%s] modular info", name)
+		log.Panicf("not registered [%s] modular info", name)
```
Comment (will-2012) [inline base/gfspapp/gfsp_modmgr.go:86]: modular->module.

Comment (sysvm) [reply base/gfspapp/gfsp_modmgr.go:86]: fixed

**Pull request merged**